--- phpunit-3.7.35/phpunit.php~	2015-11-10 16:39:54.000000000 +0200
+++ phpunit-3.7.35/phpunit.php	2015-11-10 16:39:57.907131109 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env php
+#!/usr/bin/php
 <?php
 /* PHPUnit
  *
@@ -37,10 +37,6 @@
 
 define('PHPUnit_MAIN_METHOD', 'PHPUnit_TextUI_Command::main');
 
-if (strpos('@php_bin@', '@php_bin') === 0) {
-    require dirname(__FILE__) . DIRECTORY_SEPARATOR . 'PHPUnit' . DIRECTORY_SEPARATOR . 'Autoload.php';
-} else {
-    require '@php_dir@' . DIRECTORY_SEPARATOR . 'PHPUnit' . DIRECTORY_SEPARATOR . 'Autoload.php';
-}
+require '/usr/share/pear/PHPUnit/Autoload.php';
 
 PHPUnit_TextUI_Command::main();
